### ISSUE TITLE
dryrun doesn't work

### ISSUE THREAD

**Issue by dpark01** [Asymmetrik/broad-tsv-converter#1]
`node src/main.js -t` still results in "Connecting to FTP client..." in the output

`node src/main.js --dryrun` manages to say "Skipping Connection to FTP Client", however this is soon followed by "Starting polling, waiting 3000 ms to check submission status" and "Getting list of reports from FTP"

**Comment by lu-c** [comment]
Thanks, I pushed a commit that addreses the -t option not being respected

If an uploadFolder is included as a parameter, it will skip "connecting to the FTP" and run the sample-report.xml instead (in test mode). I used [this file](https://www.ncbi.nlm.nih.gov/viewvc/v1/trunk/submit/public-docs/biosample/samples/report.xml?view=markup) as a sample for how to process the reports, but if the FTP isn't connected, we can't run this on the actual response.

Does that resolve the issue?
